src/Http/Requests/FileListingRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace EscolaLms\Files\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class FileListingRequest extends FormRequest
+{
+    /**
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
this relates to #6 yet i think it's a good place to put 
https://github.com/spatie/laravel-permission#what-it-does

```php
$user->can('list files') 
```

Reply:
I'll leave that for separate issue